packages/bloc/lib/src/bloc.dart

```diff
@@ -25,9 +25,9 @@ abstract class Bloc<Event, State> extends Cubit<State>
   /// The current [BlocObserver].
   static BlocObserver observer = BlocObserver();
 
-  final _eventController = StreamController<Event>.broadcast();
+  late final _eventController = StreamController<Event>.broadcast();
 
-  StreamSubscription<Transition<Event, State>> _transitionSubscription;
+  StreamSubscription<Transition<Event, State>>? _transitionSubscription;
```

Comment:
Can this not be late? We know that we are immediately setting it in `_bindEventsToStates` in the constructor.
```suggestion
  late StreamSubscription<Transition<Event, State>> _transitionSubscription;
```